README.md

```diff
@@ -93,7 +39,11 @@ make init
 
 | Project                                                      | Description                                                  |
 | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| [AEP SDK Sample App for Android](https://github.com/adobe/aepsdk-sample-app-android) | Contains Android sample app for the AEP SDK.                 |
+| [AEP Core Extensions](https://github.com/adobe/aepsdk-core-android) | The AEPCore represents the foundation of the Adobe Experience Platform SDK.                |
```

Comment:
- | Core extensions | The Mobile Core represents the foundation of the Adobe Experience Platform mobile SDK. |
- | Edge Network extension | The Edge Network extension allows you to send data to the Adobe Edge Network from a mobile application. |
- | Consent for Edge Network extension | The Consent ...
- | Assurance extension (link should be https://github.com/adobe/aepsdk-assurance-android) | The Assurance extension enables validation workflows for your SDK implementation.
- | AEP SDK sample app for Android | Contains the Android sample app for the AEP SDKs.